# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../ebtables/ebtables-2.0.10-2-remove-unused-ret-var.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ebtables-v2.0.10-2/communication.c.orig	2011-11-28 00:04:02.936104275 +0100
+++ ebtables-v2.0.10-2/communication.c	2011-11-28 00:07:17.076104276 +0100
@@ -256,7 +256,7 @@
 
 static int store_counters_in_file(char *filename, struct ebt_u_replace *repl)
 {
-	int size = repl->nentries * sizeof(struct ebt_counter), ret = 0;
+	int size = repl->nentries * sizeof(struct ebt_counter);
 	unsigned int entries_size;
 	struct ebt_replace hlp;
 	FILE *file;
@@ -272,13 +272,11 @@
 	   sizeof(unsigned int) ||
 	   fseek(file, entries_size + sizeof(struct ebt_replace), SEEK_SET)) {
 		ebt_print_error("File %s is corrupt", filename);
-		ret = -1;
 		goto close_file;
 	}
 	if (fwrite(repl->counters, sizeof(char), size, file) != size) {
 		ebt_print_error("Could not write everything to file %s",
 				filename);
-		ret = -1;
 	}
 close_file:
 	fclose(file);
